examples/compiler/00_index.ipynb

```diff
@@ -0,0 +1,53 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Devito's Operator \n",
+    "\n",
+    "In Devito, an `Operator` carries out three fundamental tasks: generation of low-level code, JIT compilation, and execution. Overall, the lowering process performed by an `Operator` – from high-level equations to dynamically compiled and executable code – consists of multiple compiler passes, summarized in the Figure bellow. \n",
+    "\n",
+    "![Compiler Stages](https://i.ibb.co/dKjsZ72/compiler-stages.png)\n",
+    "\n",
+    "The following series of notebooks will expose key aspects of those compilation stages in a hands-on fashion. We will be exploring the heart of the `Operator` class, which can be found at `devito/operator.py`.\n",
+    "\n",
+    "The examples and exercices shown here are mostly taken either from tests found in the folder `devito/tests`, or from the documentation itself.\n",
+    "\n",
+    "## Outline\n",
+    "- 01) Lowering\n",
+    "    - Indexification\n",
+    "    - Substitution\n",
```

Comment:
Is this a different notebook than indexification?

again, here you could add links to the actual notebooks through the usual syntax \[item\](link)

Reply:
no, i thought of it as in the same notebook

Reply:
about these links, i'm not sure how to proceed regarded that they are not in mater yet, so the shortest way i found was to add the link for my this current branch. like:

`01) [Lowering](https://github.com/opesci/devito/blob/master/examples/compiler/01_iet.ipynb)`

Reply:
no worries, you can add them in a later PR otherwise

Reply:
see my comment below for more details -- but basically we can infer what the link names will be

Reply:
ok